src/api/spec/features/webui/requests/submissions_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'browser_helper'
+
+RSpec.feature 'Bootstrap_Requests_Submissions', type: :feature, js: true, vcr: true do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'madam_submitter') }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package_with_file, name: 'Quebec', project: source_project) }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'mr_receiver') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package_with_file, name: 'Quebec', project: target_project) }
+  let(:bs_request_description) { 'Youpi!' }
+
+  context 'submit package' do
+    describe 'setting a target package' do
+      scenario 'creates a BsRequest with target package name' do
+        login submitter
+        visit package_show_path(source_project, source_package)
+        click_link('Submit Package')
+        fill_in('To target project:', with: target_project.name)
+        fill_in('To target package:', with: target_package.name)
+        fill_in('bs_request_description', with: bs_request_description)
+        expect { click_button('Submit') }.to change(BsRequest, :count).by(1)
```

Comment:
IMO, I would not check if the database here, for that we have controller and model tests. Here I would just test the workflow. That clicking on `Submit` I will end up to request show page.